static/src/stylesheets/module/onward/_feedback.scss

```diff
@@ -0,0 +1,55 @@
+
+.feedback__form {
+    border: 0px;
+    outline: none;
+    display: none;
+}
+
+.feedback__form--selected {
+    display: block;
+}
+
+.feedback__element {
+    margin-bottom: 1.5rem;
+    margin-top: 1.5rem;
+}
+
+.feedback__entry {
+    border: 1px solid #dcdcdc;
+    padding: .75rem;
+    outline: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border-radius: .5rem;
+    display: block;
+    width: 80%;
+}
+
+.feedback__textarea {
+    height: 13em;
+}
+
+.feedback__select {
+    border: 1px solid #dcdcdc;
+    padding: .5rem;
+    margin-top: 1.5em;
+    outline: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border-radius: 1.2rem;
+    display: block;
+    cursor: pointer;
+    background: #ffffff url(//jobs.theguardian.com/assets/dist/images/down-arrow.png) no-repeat 93% 50%;
```

Comment:
It would be better to use an SVG for this, if possible. It is likely we already have a down arrow in the [icon directory](https://github.com/guardian/frontend/tree/master/static/src/inline-svgs/icon)

We have [documentation](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md) on how to inline SVGs. Let me know if you need a hand.